packages/@o3r/core/builders/run-script/index.ts

```diff
@@ -0,0 +1,29 @@
+
+import { BuilderOutput, createBuilder } from '@angular-devkit/architect';
+import { getPackageManagerRunner } from '@o3r/schematics';
+import { execSync } from 'node:child_process';
+import { join, resolve } from 'node:path';
+import * as fs from 'node:fs';
+import type { RunScriptBuilderSchema } from './schema';
+
+export default createBuilder<RunScriptBuilderSchema>(async (options, context): Promise<BuilderOutput> => {
+  context.reportRunning();
+  context.reportProgress(1, 1, `Running ${options.script} !!!`);
+  const specifiedRoot = context.target?.project && (await context.getProjectMetadata(context.target.project)).root?.toString();
+  const projectRoot = specifiedRoot ? resolve(context.workspaceRoot, specifiedRoot) : context.currentDirectory;
+  const angularJsonPath = join(context.workspaceRoot, 'angular.json');
+  const angularJson = JSON.parse(fs.readFileSync(angularJsonPath).toString());
```

Comment:
It is not something you can get from context ?

Reply:
I could not find anything in context which exports it :(

Reply:
:'(
In that case you should check the if file exist and warn if it does not exist

Reply:
Indeed. I have done the update.